src/utils/bitcoin-core-client.ts

```diff
@@ -3,7 +3,9 @@
 import { MonetaryAmount } from "@interlay/monetary-js";
 import Big from "big.js";
 import { WrappedCurrency } from "../types";
+// import { createRequire } from "module";
 
+// const require = createRequire(import.meta.url);
```

Comment:
Is this needed to be commented out here?

Reply:
Good catch, not needed anymore. That is leftover from an earlier attempt to use `require` inside esm.